server/src/main/java/io/crate/execution/engine/aggregation/impl/CountAggregation.java

```diff
@@ -261,4 +284,51 @@ public LongState removeFromAggregatedState(RamAccounting ramAccounting,
         }
         return previousAggState;
     }
+
+    @Override
+    public DocValueAggregator<?> getDocValueAggregator(List<DataType<?>> argumentTypes,
+                                                       List<MappedFieldType> fieldTypes) {
```

Comment:
Could we also optimize the string type case?

Reply:
Yes, I was looking into it because of the rest `StringAgg`, `ArrayAgg` and `CollectAgg`, I was planning to do it as a separate step, but can add it here as well.

Reply:
Also fine to follow up on it later.